internal/server/data/migrations_test.go

```diff
@@ -1,231 +1,363 @@
 package data
 
 import (
+	"database/sql"
+	"fmt"
 	"io/ioutil"
-	"os"
 	"path/filepath"
+	"runtime"
 	"testing"
+	"time"
 
-	gocmp "github.com/google/go-cmp/cmp"
 	"github.com/rs/zerolog"
 	"gorm.io/gorm"
 	"gotest.tools/v3/assert"
 	"k8s.io/utils/strings/slices"
 
 	"github.com/infrahq/infra/internal/logging"
+	"github.com/infrahq/infra/internal/server/data/migrator"
 	"github.com/infrahq/infra/internal/server/models"
 	"github.com/infrahq/infra/internal/testing/patch"
 )
 
-func TestMigration_SettingsPopulatePasswordDefaults(t *testing.T) {
```

Comment:
This test case can't be represented in the table test yet, because we don't have an explicit migration for this. I'll be restoring it in the follow up PR when we add an explicit migration.